src/components/details/details.yaml

```diff
@@ -1,3 +1,37 @@
+params:
+- name: summaryText
+  type: string
+  required: true
```

Comment:
Are we consistent with this throughout? We say it's `required` if you have to pass either one or the other?

Reply:
Yes 👍 See this in PR description:
> Regarding the "or" relationship between text and html attributes, we are continuing to explain this in the description of the attributes, ie. "If html is set, this is not required." There didn't seem to be a sensible way of trying to show this relationship in a simple table so using just text appeared to be the most accessible method.